Document AWS Gov cloud STS authentication (#1113)
spectrocloud/librarium: content/docs/04-clusters/01-public-cloud/01-aws/05-add-aws-accounts.md

<Tabs.TabPane tab="Using the Access Credentials Method" key="Using the Access Credentials Method">

> To add an AWS cloud account using static access credentials follow the steps below:

Reviewer: ```suggestion
To add an AWS cloud account using static access credentials follow these steps:
```

> You can validate the account is available in Palette by reviewing the list of cloud accounts. To review the list of cloud accounts navigate to the left **Main Menu**. Click on **Tenant Settings**. Next, click on **Cloud Accounts**. Your newly added AWS cloud account is listed under the AWS sections.

Reviewer: ```suggestion
You can validate the account is available in Palette by reviewing the list of cloud accounts. To review the list of cloud accounts navigate to the left **Main Menu**. Click on **Tenant Settings**. Next, click **Cloud Accounts**. Your newly added AWS cloud account is listed under the AWS sections.
```
